app/webpacker/packs/application.js

```diff
@@ -15,10 +15,20 @@ $(document).ready(function() {
   $('#markdown-editor').keyup(function() {
     createPreview('#markdown-editor','#markdown-render');
   });
-
+  
   //clipboard
   $('#clipboard_copier').click(function() {
     copyToClipboard();
   });
+
+  document.querySelectorAll('.nhsuk-card--clickable').forEach((panel) => {
```

Comment:
Worth nothing that this bit of JS is written differently to the  functions above it.

- Use of ES6 arrow syntax vs ES5 function declaration.
- Use of `querySelector` which supports IE 9 +

It'll be nice to be consistent and figure out if support for versions of IE < 9

/cc @marcrice09 @guy-roberts